repo/raja-perf/application.py

```diff
@@ -10,18 +10,68 @@
 
 class RajaPerf(ExecutableApplication):
     """RAJA Performance suite"""
+
     name = "raja-perf"
 
-    tags = ['asc','single-node','sub-node','structured-grid',
-            'atomics','simd','vectorization','register-pressure',
-            'high-memory-bandwidth','regular-memory-access',
-            'mpi','network-point-to-point','network-latency-bound',
-            'c++','raja','cuda','hip','openmp','sycl']
+    tags = [
+        "asc",
+        "single-node",
+        "sub-node",
+        "structured-grid",
+        "atomics",
+        "simd",
+        "vectorization",
+        "register-pressure",
+        "high-memory-bandwidth",
+        "regular-memory-access",
+        "mpi",
+        "network-point-to-point",
+        "network-latency-bound",
+        "c++",
+        "raja",
+        "cuda",
+        "hip",
+        "openmp",
+        "sycl",
+    ]
+
+    executable(
+        "run",
+        template=["{execute}" + " --size {size}" + " --repfact {repfact}"],
+        use_mpi=True
+    )
+
+    # executable(
+    #     "run",
+    #     template=["{execute}" + " --size {size}" + " --repfact {repfact}" + " {additional_args}"],
+    #     use_mpi=True
+    # )
+
+    workload("suite", executables=["run"])
+
+    # workload_variable(
+    #     "additional_args",
+    #     default="--variants {variants} --tunings {tunings}",
+    #     workloads=["suite"],
+    #     description="",
+    # )
+
+    # workload_variable("variants", default="", workloads=["suite"], description="")
 
-    executable('run', 'raja-perf.exe', use_mpi=True)
+    # workload_variable("tunings", default="", workloads=["suite"], description="")
```

Comment:
I would like to have the option for `--variants` and `--tunings` instead of having to manually edit the `execute_experiment` files. The problem with this code is that when no argument is provided, the arguments are still injected, e.g. `raja-perf.exe --repfact 5 ... --variants --tunings`, when ideally they should not appear if not provided `raja-perf.exe --repfact 5 ...`

This can be done by checking in `experiment.py`, but that code results in the arguments injecting into `experiment_run_dir` (e.g. `...raja-perf.exe/raja-perf_suite_strong_1_33554432_5 --variants RAJA_CUDA --tunings block_256 ...`) since `execute` is an experiment variable.